packages/sdk-react/src/components/auth/Auth.tsx

```diff
@@ -199,10 +205,11 @@ const Auth: React.FC<AuthProps> = ({
         }
       }
 
+      const iframePublicKey = await authIframeClient!.initEmbeddedKey();
```

Comment:
Each of these checks could be considered overkill, but at the same time I don't think it would hurt. Unless there's a flow/edge case where the `authIframeClient.iframePublicKey` can be inconsistent across different components?

However, this `useEffect` should help mitigate against that

```
useEffect(() => {
    if (authIframeClient) {
      setComponentReady(true);

      const updateIframeKey = async () => {
        await authIframeClient.initEmbeddedKey();
      };

      updateIframeKey();
    }
  }, [authIframeClient]);
```